OpenRA.Game/WorldViewportSizes.cs

```diff
@@ -0,0 +1,33 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA
+{
+	public class WorldViewportSizes : IGlobalModData
+	{
+		public readonly int2 CloseWindowHeights = new int2(480, 600);
+		public readonly int2 MediumWindowHeights = new int2(600, 900);
+		public readonly int2 FarWindowHeights = new int2(900, 1300);
+
+		public readonly float MaxZoomScale = 2.0f;
+		public readonly int MaxZoomWindowHeight = 240;
+		public readonly bool AllowNativeZoom = true;
```

Comment:
Can we please make these settings yaml configurable?

Reply:
They already are. `IGlobalModData` implementations are mapped to the blocks in `mod.yaml` - I didn't add any overrides because the defaults here are (AFAIK) suitable for all the default mods.